### ISSUE TITLE
Partial dereference

### ISSUE THREAD

**Issue by zoellner** [APIDevTools/json-schema-ref-parser#114]
Would it be possible to somehow allow to bundle a schema and only partially dereference it?

My use case is an OpenAPI definition that I am currently dereferencing to make it work with AWS API Gateway. I mainly need to dereference everything in the components/schemas section but would like to keep internal references in the paths section in place. 

Bundle doesn't quite work as expected in this case when the schemas contain nested references. 

So basically I want to bundle the main file but dereference every external $ref before bundling it into the main file.

**Comment by zoellner** [comment]
Thanks. That works.
